src/test/scala/edu/berkeley/ce/rockslicing/BlockSpec.scala

```diff
@@ -637,30 +643,32 @@ class BlockSpec extends FunSuite {
 
   test("Volume of two cube with extra plane should should be 4.0") {
     val redundantBoundingFaces = boundingFaces3 ++ List(
-      Face((-1.0/math.sqrt(3.0), -1.0/math.sqrt(3.0), -1.0/math.sqrt(3.0)), 0.0, phi=0, cohesion=0)
+      Face(Array(-1.0/math.sqrt(3.0), -1.0/math.sqrt(3.0), -1.0/math.sqrt(3.0)), 0.0, phi=0, cohesion=0)
     )
-    val redundantTwoCube = Block((0.0, 0.0, 0.0), redundantBoundingFaces)
-    val nonRedundantTwoCube = Block((0.0, 0.0, 0.0), redundantTwoCube.nonRedundantFaces)
+    val redundantTwoCube = Block(Array(0.0, 0.0, 0.0), redundantBoundingFaces)
+    // TODO this causes the test to fail, but shouldn't it?
```

Comment:
Okay, I've removed them for now, but let me know if you'd prefer a different approach.